src/components/Topbar/Topbar.tsx

```diff
@@ -1,15 +1,8 @@
 import routes, { QUERY_PARAMS } from '@/config/routes'
 import { useLocation, useNavigate } from 'react-router-dom'
 import React, { useEffect, useState } from 'react'
-import {
-  FullLogo,
-  Header,
-  LogoLink,
-  NavigationContainer,
-  SearchbarContainer,
-  ShortLogo,
-  StyledSearchbar,
-} from './TopNavbar.style'
+import { TopbarBase } from '@/shared/components'
+import { SearchbarContainer, StyledSearchbar } from './Topbar.style'
 import { RoutingState } from '@/types/routing'
 
 const TopNavbar: React.FC = () => {
```

Comment:
I think if we have `StudioTopbar`, we probably should call this `ViewerTopbar`. Also let's make sure the component name matches the filename